cucumber/api/features/authenticate.feature

```diff
@@ -15,11 +16,23 @@ Feature: Exchange a role's API key for a signed authentication token
     """
       <86>1 * * conjur * authn
       [auth@43868 authenticator="authn"]
-      [subject@43868 role="cucumber:user:alice"]
+      [subject@43868 role="cucumber:user:alice" user="alice"]
```

Comment:
@egvili @netacoral is there a requirement for this to be `user` and not `username`? would we prefer this to be `username`?

Reply:
As far as I understand, `user` is the field being queried by other tools (e.g. our SaaS solution). So it has to be user

Reply:
@orenbm `user ` is already in use in other audit events, so the requirement was to add it to these events as well.